### PR TITLE
added new make commands

### PR DESCRIPTION
**Pull request by roberthocking**
# Description of the issue
Added 3 new make commands

# Description of changes
Added make commands to deploy and destroy and EKS cluster as well as a command to deploy a test image of the agent using a helm chart

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
Ran all commands locally

### CODE REVIEW

**Comment by bdrennz** [comment]
This looks good to me. Can we update the `README.md` as well?

**Review by bdrennz — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -271,4 +271,24 @@ dockerized-build-amd64:
 
 # Use vendor instead of proxy when building w/ vendor folder
 dockerized-build-vendor:
-	$(DOCKER_BUILD_FROM_SOURCE) --build-arg GO111MODULE=off .
\ No newline at end of file
+	$(DOCKER_BUILD_FROM_SOURCE) --build-arg GO111MODULE=off .
+
+.PHONY: create-eks-cluster
+create-eks-cluster:
+	@eksctl create cluster \
```
Comment (dmepham) [inline Makefile:278]: just curious, but does this choose the default VPC to launch in and any subnets in those zones within that VPC?

File: Makefile [hunk 2]
```diff
@@ -271,4 +271,24 @@ dockerized-build-amd64:
...
+		--name $(namespace) \
+		--region us-east-1 \
```
Comment (dmepham) [inline Makefile]: is it safe to assume that we're always deploying to us-east-1?

File: Makefile [hunk 3]
```diff
@@ -271,4 +271,24 @@ dockerized-build-amd64:
...
+		--zones us-east-1a,us-east-1b
+
+.PHONY: destroy-eks-cluster
+destroy-eks-cluster:
+	@eksctl delete cluster --region=us-east-1 --name=$(namespace)
+
+.PHONY: deploy-test
+deploy-test:
```
Comment (dmepham) [inline Makefile:288]: eksctl creates a kubeconfig file for you, right? so you could run `make create-eks-cluster` and `make deploy-test` one after another without any extra setup?

**Review by dmepham — COMMENTED** [review]
looks great to me!

File: Makefile (hunk 3 above)
Comment (bdrennz) [reply Makefile:288]: Ooo that's a good point. Right, maybe you can check the kube context on the machine before running the command.

File: Makefile (hunk 2 above)
Comment (roberthocking) [reply Makefile]: for our use cases, yes it is safe. i'll fix it up and add region as a parameter just in case

**Pull request merged**